dhcp-relay/test.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash -x
+
+OUTER_VLAN=83
+INNER_VLAN=20
+UPLINK_VLAN=84
+IF="ens6f0"
```

Comment:
It should be possible to test this using network namespaces; having a test that is runnable without needing special hardware makes it way easier to test. Such a test should really be added so it can be run as part of a CI job.